india_banking_connector/connectors/doctype/idfc_connector/idfc_connector.py

```
@@ -0,0 +1,415 @@
+		}
+
+		data = {
+			"grant_type": "client_credentials",
+			"scope": self.scope,
+			"client_id": self.get_password("client_id"),
+			"client_assertion_type": "urn:ietf:params:oauth:client-assertion-type:jwt-bearer",
+			"client_assertion": self.get_client_assertion(),
+		}
+
+		try:
+			response = requests.post(
```

Ruff (0.15.0)

[error] Probable use of `requests` call without timeout

(S113)